player_tank.py

```diff
@@ -40,9 +40,9 @@ def update_me(self,player_pos):
 	# 2. The methods where you define all the class functions (methods)
     def draw_me(self):
         copied_image_bottom = self.image_bottom.copy()
-        pygame.transform.rotate(copied_image_bottom, self.bottom_angle)
-        self.screen.blit(self.image_bottom, [self.x,self.y])
-        # self.screen.blit(self.image_top, [self.x,self.y])
+        rot_image_bottom = pygame.transform.rotate(copied_image_bottom, self.bottom_angle)
```

Comment:
Have you tried using a function like this one?
[RotateCenter](https://www.pygame.org/wiki/RotateCenter)